Cargo.toml

```diff
@@ -3,26 +3,26 @@ name = "yew_webview_bridge"
 description = "2-way communcation bridge between web-view and yew"
 keywords = ["yew", "web-view"]
 license = "MIT OR Apache-2.0"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Could you please remove the version bump? I'll bump it when I release a new version.